source/ptreferential/ptreferential_ng.cpp

```diff
@@ -365,6 +372,16 @@ struct Eval : boost::static_visitor<Indexes> {
         return res;
     }
     Indexes operator()(const ast::Expr& expr) const { return boost::apply_visitor(*this, expr.expr); }
+
+private:
+    // helper to add required param to methods since(), until() and between().
+    static size_t nb_extra_args_between(const type::Type_e& type) {
```

Comment:
Why static ?

Reply:
don't need `static`. It is just a private function

Reply:
Yes, just add `const`